### ISSUE TITLE
Enforce in-order delivery of messages

### ISSUE THREAD

**Issue by 07151129** [poljar/weechat-matrix#171]
If a message containing newlines is pasted, it gets sent as multiple messages: one message per line. As these messages are sent in parallel, they often arrive at the recipient reordered. It would be better to have an option to enforce their in-order delivery: do not submit the next message until the current one has been delivered.

I am aware this is better solved by not splitting multiline messages at all, but there seems to be no progress with markdown support, so as a workaround the proposed feature would be helpful.

**Comment by trygveaa** [comment]
Probably easier to just fix multiline, most of the changes in the markdown branch is not needed for that.

**Comment by 07151129** [comment]
I would still find the proposed option useful regardlessly of multiline support.
